playground/tanka/lib/apps/aperture-grafana/main.libsonnet

```diff
@@ -16,7 +16,7 @@ local dataSources =
         access: 'proxy',
         url: 'http://controller-prometheus-server',
         jsonData: {
-          timeInterval: '1s',
+          timeInterval: '5s',
```

Comment:
Shouldn't this be equal to scrape interval?

Reply:
Yes, addressed in [10837c8](https://github.com/fluxninja/aperture/pull/2168/commits/10837c833da8a2c9d93e7e3bf17f23f800df8918)